packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -240,6 +240,34 @@ export default declare((api, opts) => {
     }
   }
 
+  function hasMoreThanOneBinding(node) {
+    // ArrayPattern
+    if (node.elements) {
+      const nonNullElements = node.elements.filter(element => element !== null);
+      if (nonNullElements.length > 1) return true;
+      else return hasMoreThanOneBinding(nonNullElements[0]);
+      // ObjectPattern
+    } else if (node.properties) {
+      if (node.properties.length > 1) return true;
+      // Case of `const {} = {}`. This shouldn't be hit as
+      // we currently call `hasMoreThanOneBinding` only under visitRestElements.
```

Comment:
```suggestion
```

Per https://github.com/babel/babel/pull/13711/files#r701061273

Reply:
Included this in the latest commit.